packages/trader/src/AppV2/Hooks/__tests__/useOrderDetails.spec.tsx

```diff
@@ -0,0 +1,158 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { CONTRACT_TYPES, TContractInfo } from '@deriv/shared';
+import useOrderDetails from '../useOrderDetails';
+
+jest.mock('@deriv/translations', () => ({
+    localize: jest.fn(text => text),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    CONTRACT_TYPES: {
+        TURBOS: { LONG: 'turbos_long', SHORT: 'turbos_short' },
+        MULTIPLIER: { DOWN: 'multiplier_down', UP: 'multiplier_up' },
+        MATCH_DIFF: { MATCH: 'match_diff_match', DIFF: 'match_diff_diff' },
+        EVEN_ODD: { EVEN: 'even_odd_even', ODD: 'even_odd_odd' },
+        OVER_UNDER: { OVER: 'over_under_over', UNDER: 'over_under_under' },
+        RESET: { CALL: 'reset_call' },
+        PUT: 'put',
+        CALLE: 'calle',
+        CALL: 'call',
+        TOUCH: { ONE_TOUCH: 'touch_one_touch', NO_TOUCH: 'touch_no_touch' },
+        ACCUMULATOR: 'accumulator',
+        VANILLA: { CALL: 'vanilla_call', PUT: 'vanilla_put' },
+    },
+    getDurationPeriod: jest.fn(),
+    getDurationTime: jest.fn(),
+    getDurationUnitText: jest.fn(),
+    getGrowthRatePercentage: jest.fn(() => '10'),
+    isAccumulatorContract: jest.fn(),
+    isResetContract: jest.fn(),
+    addComma: jest.fn(),
+}));
+
+jest.mock('App/Components/Elements/PositionsDrawer/helpers', () => ({
+    getBarrierValue: jest.fn(),
+}));
+
+const mockData: TContractInfo = {
```

Comment:
We can use `mockContractInfo` function for mocking. Not only in this test file, but in others too